bin/node/runtime/Cargo.toml

```diff
@@ -68,7 +68,7 @@ pallet-balances = { version = "4.0.0-dev", default-features = false, path = "../
 pallet-bounties = { version = "4.0.0-dev", default-features = false, path = "../../../frame/bounties" }
 pallet-child-bounties = { version = "4.0.0-dev", default-features = false, path = "../../../frame/child-bounties" }
 pallet-collective = { version = "4.0.0-dev", default-features = false, path = "../../../frame/collective" }
-pallet-contracts = { version = "4.0.0-dev", default-features = false, path = "../../../frame/contracts" }
+pallet-contracts = { version = "4.0.0-dev", default-features = false, path = "../../../frame/contracts", features = ["unsafe-debug"] }
```

Comment:
Is there a reason why we set this here? Isn't it covered by tests + CI anyways?

Reply:
it's the only configuration that allowed me to pass both commands (which are triggered in CI):
 - `SKIP_WASM_BUILD=1 cargo check --locked --benches --all -p kitchenskin-runtime`
 - `cargo check --locked --benches --all -p kitchenskin-runtime`

Reply:
That would be very had as it means the pallet requires that feature. For me it works without that feature. You need to remove `type Debug = ()` of course.

Reply:
it's failing though: https://gitlab.parity.io/parity/mirrors/substrate/-/jobs/3331887

that is really pain in the back and I'm struggling with it from the very beginning; no idea why it requires this `Debug` type here

Reply:
Ahh yeah of course because we manually enable the feature there for this crate. Yeah you have to roll back this change :)

Reply:
@athei while the two commands are now passing for me as well, for some reason, rust doc is requiring this item - you can run:
```
cargo doc --locked --workspace --all-features --no-deps
```

Nothing like:
```rust
#[cfg(doc)]
type Debug = ();
// or
#[cfg(feature = "unsafe-debug")]
type Debug = ();
```
helps

Reply:
You didn't roll back the change yet.

Reply:
I think I did - there's no change to `/bin/node` directory